src/Infrastructure/Connector/Shopware6Connector.php

```diff
@@ -133,12 +133,20 @@ private function resolveResponse(ResponseInterface $response): ?string
 
     private function logRequest(string $uid, ActionInterface $action): void
     {
+        $requestMethod = $action->getRequest()->getMethod();
+        $requestPath = $action->getRequest()->getUri()->getPath();
+
+        $this->logger->info(sprintf('Shopware6 %s: %s',
```

Comment:
To jest juz logowane ponizej.

Reply:
info jes wyswietlane tylko przy very verbose -vv -> mniej danych bez response'a